embedding-calculator/srcext/rude_carnie/README.md

```diff
@@ -0,0 +1,227 @@
+Rude Carnie: Age and Gender Deep Learning with TensorFlow
```

Comment:
We need to ask for permission or license before adding their code to our repository...

Reply:
Asked - https://github.com/dpressel/rude-carnie/issues/109

Reply:
Removed this code until we get permissions